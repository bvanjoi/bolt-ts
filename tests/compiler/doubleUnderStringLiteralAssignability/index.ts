// From `github.com/microsoft/TypeScript/blob/v5.8.2/tests/cases/compiler/doubleUnderStringLiteralAssignability.ts`, Apache-2.0 License

var shouldBeOk: '__dunder' = '__dunder';
var bad: '__dunder' = 'no_dunder';
//~^ ERROR: Type '"no_dunder"' is not assignable to type '"__dunder"'.
var okok: '___thunder' = '___thunder';
var alsoOk: '_sunder' = '_sunder';
