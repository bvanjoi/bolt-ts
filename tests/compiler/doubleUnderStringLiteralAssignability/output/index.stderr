  × Type '"no_dunder"' is not assignable to type '"__dunder"'.
   ╭─[../../tests/compiler/doubleUnderStringLiteralAssignability/index.ts:4:5]
 4 │ bad
   · ───
   ╰────
